source/utilities.tex

```diff
@@ -19886,6 +19886,30 @@
 For floating-point types this field specifies the formatting precision.
 For string types it specifies how many characters will be used from the string.
 
+\pnum
+When the \tcode{L} option is used, the form used for the conversion
+is called the \defn{locale-specific} form.
+The \tcode{L} option is only valid for arithmetic types, and
+its effect depends upon the type.
+\begin{itemize}
+
+\item
+For integral types, the locale-specific form
+causes the context’s locale to be used
+to insert the appropriate digit group separator characters.
+
+\item
+For floating-point types, the locale-specific form
+causes the context’s locale to be used
+to insert the appropriate digit group and decimal radix separator characters.
```

Comment:
The examples in the paper show the "decimal" radix separator character to be used even with hexfloats. Something doesn't add up here.

Reply:
@jensmaurer This was a drafting error on my part and I'm sorry it didn't get picked up.

The examples are correct and the wording is is not correct.  I suggest `s/decimal//`.

The intention is to give access the features currently available when using iostreams, e.g.

```c++
#include <iostream>
#include <locale>

int main() {
    const std::locale gb{"en_GB"};
    std::cout.imbue(gb);
    std::cout << std::hex << 123456 << "\n";
    return 0;
}
```
which produces `1e,240` in implementations I have access to right now.

Reply:
@zygoloid, are you ok with me making this normative change editorially?

Reply:
@burblebee @zygoloid If necessary, I will raise an LWG issue.

Reply:
I think the intent is sufficiently clear, especially given the description given in the paper.